types/wojtekmaj__react-datetimerange-picker/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for @wojtekmaj/react-datetimerange-picker 3.0
+// Project: https://github.com/wojtekmaj/react-datetimerange-picker (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Pesven <https://github.com/Psvensso>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import { CalendarProps } from 'react-calendar';
```

Comment:
We have no guarantee these props will be available in the future versions of React-Calendar; for now I think this might be fine, but still we would need to import this package here...?

Reply:
If the current version uses all props from calendar then we should re-use their interface. In the future, this package needs to be updated if the dependency is removed. Or else your documentation "DateTimeRangePicker component passes all props to React-Calendar, with the exception of className" is invalid and typescript users won't be able to add those props to the component.